apps/api/src/scraper/scrapeURL/transformers/uploadScreenshot.ts

```diff
@@ -0,0 +1,26 @@
+// This file is an exception to the "no supabase in scrapeURL" rule,
```

Comment:
when do we use this? fire-engine uploads screenshots on its own, is it for playwright?

Reply:
This is for scrapingbee.